### DISCUSSION TITLE
How can i upload files by using powershell command ？

### DISCUSSION THREAD

**Discussion by whxiewy** [rejetto/hfs#96] [General]
Just like the title 。 Hoping  gracious somebody tells me，thanks。

**Comment by rejetto** [comment]
i don't know powershell, but i can tell you how to do it with curl
curl -T FILE URL/

URL is pointing to the folder, and as you see must finish with /

additional note: the -T parameter uses PUT as http method

**Reply by whxiewy** [reply]
Thanks very much。I download curl for windows to achieve 。

**Comment by rejetto** [comment]
...and if you google for it you'll find how to make a PUT request with powershell
https://stackoverflow.com/questions/57127805/how-to-http-put-a-file-in-powershell-and-capture-response-body
